src/components/NcAvatar/NcAvatar.vue

```diff
@@ -490,6 +492,8 @@ export default {
 	},
 
 	beforeDestroy() {
+		unsubscribe('settings:avatar:updated', this.loadAvatarUrl)
+		unsubscribe('settings:display-name:updated', this.loadAvatarUrl)
```

Comment:
I would have used the proper context ?

```suggestion
		unsubscribe('user:avatar:updated', this.loadAvatarUrl)
		unsubscribe('user:display-name:updated', this.loadAvatarUrl)
```